packages/commonwealth/client/scripts/views/components/proposals/voting_results.tsx

```diff
@@ -38,6 +55,14 @@ export const VotingResults = (props: VotingResultsProps) => {
     };
   }, [forceRerender]);
 
+  const { data } = useAaveProposalVotesQuery({
+    moduleReady: app.chain?.network === ChainNetwork.Aave || !isLoading,
+    chainId: app.chain?.id,
+    proposalId: proposal.identifier,
+  });
+
+  const votes = data || proposal.getVotes();
```

Comment:
do not know the overall scope of this PR but curious why we need to do the fetch OR getVotes from the proposal? are they equivalent?

Reply:
This is because the react query `useAaveProposalVotesQuery` function only fetches votes for `aave` but we support votes for many other governance protocols. In this PR, I only modified how we load aave proposals/votes (`data`) so proposals/votes from other protocols must still be loaded from the proposal controllers/store (`proposal.getVotes()`).